config/common/common.go

```diff
@@ -129,6 +129,12 @@ func GetFullyQualifiedIDFn(serviceProvider string, keyPairs ...string) tjconfig.
 
 		path := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/%s", subIDStr, rgStr, serviceProvider)
 		for i := 0; i < len(keyPairs); i += 2 {
+			// https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/iothub_fallback_route#import
+			// FallbackRoute is always default and we don't have associated parameter for it
+			if keyPairs[i] == "FallbackRoute" {
+				path = filepath.Join(path, "FallbackRoute", "default")
+				return path, nil
```

Comment:
What if in another context `FallbackRoute` is not the last path component or its value is not a constant but instead a value of a parameter? Please see the suggestion below for the configuration of the `azurerm_iothub_fallback_route` resource.